examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uyn2xrR7R1K_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "exp_name = 'quartznet3x1_an4'\n",
+        "work_dir = './myExps/'\n",
+        "neural_factory = nemo.core.NeuralModuleFactory(\n",
+        "    log_dir=work_dir+\"/as4_logdir/\",\n",
+        "    checkpoint_dir=\"./myExps/checkpoints/\" + exp_name,\n",
+        "    create_tb_writer=True,\n",
+        "    random_seed=42,\n",
+        "    tensorboard_dir=work_dir+'/tensorboard/',\n",
+        ")"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "k-juqc40p8KN",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now that we have our neural module factory, we can specify our **neural modules and instantiate them**. Here, we load the parameters for each module from the configuration file. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "mC-KPOy-rpLA",
+        "colab_type": "code",
+        "outputId": "1d902505-6e35-4eb8-aebf-8401c1bdd39c",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "logging = nemo.logging\n",
+        "yaml = YAML(typ=\"safe\")\n",
+        "with open('../configs/quartznet_spkr_3x1x512_xvector.yaml') as f:\n",
+        "    spkr_params = yaml.load(f)\n",
+        "\n",
+        "sample_rate = spkr_params[\"sample_rate\"]\n",
+        "time_length = spkr_params.get(\"time_length\", 8)\n",
+        "logging.info(\"max time length considered for each file is {} sec\".format(time_length))"
+      ],
+      "execution_count": 20,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[NeMo I 2020-05-11 02:21:15 <ipython-input-20-4537deb08e36>:8] max time length considered for each file is 8 sec\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5VgzNS1lrrqS",
+        "colab_type": "text"
+      },
+      "source": [
+        "Instantiating train data_layer using config arguments. `labels = None` automatically creates output labels from manifest files, if you would like to pass those speaker names you can use the labels option. So while instatilatin eval data_layer we can use data_layer labels as it should both match to same speaker output labels. This comes handy while training on multiple datasets with more than one manifest file. "
```

Comment:
`So while instatilatin eval data_layer we can use data_layer labels as it should both match to same speaker output labels.` doesn't make sense to me. Please fix.

This comes in handy

Reply:
Done

Reply:
`So while instantiating eval data_layer we can use data_layer labels to match same speaker output labels as train. `
should be
`So while instantiating eval data_layer, we can use pass labels to the class in order to match same the speaker output labels as we have in the training data layer. `